_pages/general/visas.md

```diff
@@ -5,17 +5,89 @@ description: How to obtain a visa, useful resources and FAQ's relating to visas
 
 # Visas
 
-If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration
-department before you commence our recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the
-482 visa.
+If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration department before you commence a recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the 482 visa.
```

Comment:
I think the immigration department changed its name, now is called `Department of Home Affairs`. Is better to keep this up-to-date. It can be a bit confusing when you arrive on the website and see a different name.

Reply:
Included in 7ea35071100cc60f45aeb4c4fd1a3dc44590f012